server/index.js

```diff
@@ -1,65 +0,0 @@
-const express = require('express')
-const bodyParser = require('body-parser')
-const cors = require('cors')
-
-const app = express()
-
-app.use( bodyParser.json() )
-app.use(cors())
-
-
-let animals = [
-    {
-      name: 'Armadillo',
-      id: '1',
-      imageUrl: 'https://upload.wikimedia.org/wikipedia/commons/thumb/b/b4/Nine-banded_Armadillo.jpg/1200px-Nine-banded_Armadillo.jpg',
-      description: ''
-    },
-    {
-      name: 'Tiger',
-      id: '2',
-      imageUrl: 'http://kids.sandiegozoo.org/sites/default/files/2017-06/animal-hero-tiger_0.jpg',
-      description: ''
-    }
-  ]
-  let id = ''
-
-//ENDPOINTS
-
-app.get('/api/animals', (req, res) => {
-    res.send(animals)
-})
-
-app.post('/api/animals', (req, res) => {
-    let newAnimal = {
-        id: id,
```

Comment:
One thing about this, is i'm pretty sure the first animal you post will have an ID that is an empty string. Since you're setting `let id = ''`, what I would maybe do is set it to be `let id = '3'` then it starts off as the first one that doesn't currently exist and increments from there. Also you have to be careful, because it is a string, the incrementing logic `i++` may not work.